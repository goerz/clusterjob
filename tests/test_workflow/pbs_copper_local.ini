[Attributes]
backend = pbs
shell = /bin/bash
rootdir = /work/goerz
workdir = cjtest

[Resources]

queue = debug
time = 00:05:00
select = 1:ncpus=32:mpiprocs=32
-A = XXXXXXXXXXXXX
-j = oe
