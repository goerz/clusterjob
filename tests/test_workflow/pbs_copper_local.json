[
  {
    "args": [
      [
        "mkdir",
        "-p",
        "/work/goerz/cjtest"
      ],
      null
    ],
    "kwargs": {
      "ignore_exit_code": false,
      "ssh": "ssh"
    },
    "response": ""
  },
  {
    "args": [
      [
        "qsub",
        "coppertest.pbs"
      ],
      null,
      "/work/goerz",
      "cjtest"
    ],
    "kwargs": {
      "ignore_exit_code": true,
      "ssh": "ssh"
    },
    "response": "436036.sdb\n"
  },
  {
    "args": [
      [
        "qstat",
        "-x",
        "436036"
      ],
      null
    ],
    "kwargs": {
      "ignore_exit_code": true,
      "ssh": "ssh"
    },
    "response": "Job id            Name             User              Time Use S Queue\n----------------  ---------------- ----------------  -------- - -----\n436036.sdb        coppertest       goerz             00:00:00 R debug           \n"
  },
  {
    "args": [
      [
        "qdel",
        "436036"
      ],
      null
    ],
    "kwargs": {
      "ignore_exit_code": true,
      "ssh": "ssh"
    },
    "response": ""
  },
  {
    "args": [
      [
        "mkdir",
        "-p",
        "/work/goerz/cjtest"
      ],
      null
    ],
    "kwargs": {
      "ignore_exit_code": false,
      "ssh": "ssh"
    },
    "response": ""
  },
  {
    "args": [
      [
        "qsub",
        "coppertest.pbs"
      ],
      null,
      "/work/goerz",
      "cjtest"
    ],
    "kwargs": {
      "ignore_exit_code": true,
      "ssh": "ssh"
    },
    "response": "436037.sdb\n"
  },
  {
    "args": [
      [
        "qstat",
        "-x",
        "436037"
      ],
      null
    ],
    "kwargs": {
      "ignore_exit_code": true,
      "ssh": "ssh"
    },
    "response": "Job id            Name             User              Time Use S Queue\n----------------  ---------------- ----------------  -------- - -----\n436037.sdb        coppertest       goerz             00:00:01 R debug           \n"
  },
  {
    "args": [
      [
        "qstat",
        "-x",
        "436037"
      ],
      null
    ],
    "kwargs": {
      "ignore_exit_code": true,
      "ssh": "ssh"
    },
    "response": "Job id            Name             User              Time Use S Queue\n----------------  ---------------- ----------------  -------- - -----\n436037.sdb        coppertest       goerz             00:00:01 R debug           \n"
  },
  {
    "args": [
      [
        "qstat",
        "-x",
        "436037"
      ],
      null
    ],
    "kwargs": {
      "ignore_exit_code": true,
      "ssh": "ssh"
    },
    "response": "Job id            Name             User              Time Use S Queue\n----------------  ---------------- ----------------  -------- - -----\n436037.sdb        coppertest       goerz             00:00:01 R debug           \n"
  },
  {
    "args": [
      [
        "qstat",
        "-x",
        "436037"
      ],
      null
    ],
    "kwargs": {
      "ignore_exit_code": true,
      "ssh": "ssh"
    },
    "response": "Job id            Name             User              Time Use S Queue\n----------------  ---------------- ----------------  -------- - -----\n436037.sdb        coppertest       goerz             00:00:01 R debug           \n"
  },
  {
    "args": [
      [
        "qstat",
        "-x",
        "436037"
      ],
      null
    ],
    "kwargs": {
      "ignore_exit_code": true,
      "ssh": "ssh"
    },
    "response": "Job id            Name             User              Time Use S Queue\n----------------  ---------------- ----------------  -------- - -----\n436037.sdb        coppertest       goerz             00:00:01 R debug           \n"
  },
  {
    "args": [
      [
        "qstat",
        "-x",
        "436037"
      ],
      null
    ],
    "kwargs": {
      "ignore_exit_code": true,
      "ssh": "ssh"
    },
    "response": "Job id            Name             User              Time Use S Queue\n----------------  ---------------- ----------------  -------- - -----\n436037.sdb        coppertest       goerz             00:00:01 R debug           \n"
  },
  {
    "args": [
      [
        "qstat",
        "-x",
        "436037"
      ],
      null
    ],
    "kwargs": {
      "ignore_exit_code": true,
      "ssh": "ssh"
    },
    "response": "Job id            Name             User              Time Use S Queue\n----------------  ---------------- ----------------  -------- - -----\n436037.sdb        coppertest       goerz             00:00:01 F debug           \n"
  }
]