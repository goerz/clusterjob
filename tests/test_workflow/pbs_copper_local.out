####################################################
Job id         : 436037.sdb
Job name       : coppertest
Workdir        : /lustre/work/goerz/cjtest
Submission Host: copper01
Compute Node   : nid00030
nid00030
nid00030
nid00030
nid00030
nid00030
nid00030
nid00030
nid00030
nid00030
nid00030
nid00030
nid00030
nid00030
nid00030
nid00030
nid00030
nid00030
nid00030
nid00030
nid00030
nid00030
nid00030
nid00030
nid00030
nid00030
nid00030
nid00030
nid00030
nid00030
nid00030
nid00030
Job started on copper07 Sun Feb 28 15:55:48 CST 2016
Current directory: /lustre/usr/local/u/goerz
####################################################
Computer node:
nid00007
Application 2665956 resources: utime ~0s, stime ~1s, Rss ~3980, inblocks ~24, outblocks ~28
Job Finished:  Sun Feb 28 15:56:49 CST 2016
